Ghidra/Framework/SoftwareModeling/src/test/java/ghidra/program/model/data/DialogResourceDataTypeTest.java

```diff
@@ -0,0 +1,210 @@
+/* ###
+ * IP: GHIDRA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ghidra.program.model.data;
+
+import static org.junit.Assert.assertEquals;
+
+import java.nio.charset.Charset;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import ghidra.program.model.address.AddressSpace;
+import ghidra.program.model.address.GenericAddressSpace;
+import ghidra.program.model.mem.ByteMemBufferImpl;
+import ghidra.program.model.mem.MemBuffer;
+import ghidra.program.model.mem.MemoryAccessException;
+import ghidra.util.LittleEndianDataConverter;
+import ghidra.util.UniversalIdGenerator;
+
+public class DialogResourceDataTypeTest {
+
+	private GenericAddressSpace addressSpace;
+
+	@Before
+	public void setUp() {
+		UniversalIdGenerator.initialize();
```

Comment:
I suspect if you derive from one of the abstract test base classes you will get the UniversalIdGenerator.initialize() for free.

Reply:
Ah yes, thank you! `AbstractGTest` did the trick.